README.md

```diff
@@ -8,3 +8,17 @@ It covers:
   -   Add files to be tracked by Git `git add 'my-file'`.
   -   Stage changes to be commited `git add 'my-file'`.
   -   Commit changes with appropriate commit message `git commit -m 'commit message'`.
+  - Finding out information regarding what files are modified and those that are in the staging area using `git status`.
+  - Printing out all the commits that are completed using `git log`.
```

Comment:
This line reflects on additional information on working with Git hence not particularly on the trunk workflow but rather as a N/B or point to note, what do you think?

Reply:
It is a helpful trick to know just in case.